pkg/util/util.go

```diff
@@ -209,10 +209,13 @@ type EXEC interface {
 }
 
 type ExecCommand struct {
+	AuthAzcopyEnv []string
 }
 
-func (ec *ExecCommand) RunCommand(cmd string) (string, error) {
-	out, err := exec.Command("sh", "-c", cmd).CombinedOutput()
+func (ec *ExecCommand) RunCommand(cmdStr string) (string, error) {
```

Comment:
this could make the function more generic:
```
RunCommand(cmdStr string, authEnv []string)
```

Reply:
```
if len(authEnv) > 0 {
  cmd.Env = append(os.Environ(), ec.AuthAzcopyEnv...)
}
```

Reply:
done